src/modules/app/routes/categories.js

```diff
@@ -0,0 +1,39 @@
+// @flow
```

Comment:
Can we have the categories.js, dislaimer.js somehow local in the routes/ directory? we could maybe put these in a Route.js in each package of a route.

Reply:
I don't understand this, sorry :D

Reply:
okay now I do, yes of course we could do that

Reply:
do we want that?

Reply:
I think this is no relevant anymore.